sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcReadBenchmark.scala

```diff
@@ -483,6 +484,64 @@ object OrcReadBenchmark {
     }
   }
 
+  def filterPushDownBenchmark(values: Int, width: Int): Unit = {
```

Comment:
Filter push-down depends on various properties of data and predicates. This is an example of filter push down performance in order to show some benefits.

Reply:
Have you seen any workload that predicate pushdown could be slower?

Reply:
Theoretically, useless predicates (selectivity 100%) only adds additional computation for both Parquet/ORC.

Reply:
Could you add a test case for useless predicates too?

Reply:
Ur, do you expect there will be much difference in some cases?
In general `most common cases`, it will be slightly slower as we can expect easily.